documentation/Gravities.md

```diff
@@ -2,16 +2,16 @@
 
 The Gravities table contains gravity measurements for sources listed in the Sources table. 
 The combination of *source*, *regime*, and *reference* is expected to be unique.
-Note that *gravity* and *regime* are strings constrained from a list of enumerated values. 
+Note that *gravity* are strings constrained from a list of enumerated values. 
 Columns marked with an asterisk (*) may not be empty.
 
 | Column Name | Description  | Unit  | Data Type | Key Type  |
 |---|---|---|---|---|
 | *source    | Unique identifier for the source |   | String(100)  | primary and foreign: Sources.source   |
 | gravity | Gravity value |  | Enumeration  |   |
-| *regime | Regime for gravity value |  | Enumeration  | primary |
+| *regime | Regime for gravity value |  |  | primary and foreign: Regimes.regime |
 | comments  | Free form comments |   | String(1000) |   |
-| *reference | Reference |   | String(30) | primary and foreign: Publications.name |
+| *reference | Reference |   | String(30) | primary and foreign: Publications.reference |
 
 Enumeraions for gravity include:
```

Comment:
What about things like red/other/intermediate/YMG/young, e.g. Liu+16.
I have just realised that may be a wider discussion beyond this PR

Reply:
Yeah, that's beyond the scope about this PR which is trying to be focused on photometry regimes.